### PR TITLE
Change to 3 clause BSD

### PR DESCRIPTION
**Pull request by wseltzer**
Derived from https://opensource.org/licenses/BSD-3-Clause
changing Copyright <YEAR> <COPYRIGHT HOLDER> to Copyright 2018 web-platform-tests contributors

Ref https://github.com/web-platform-tests/wpt/issues/11009

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,33 +1,13 @@
-# Dual-License for W3C Test Suites
+# 3-Clause BSD License
```
Comment (annevk) [inline LICENSE.md]: This PR seems nearly equivalent to #11191 except that this doesn't say "The" at the start and the "Redistribution ..." line appears modified in the diff, whereas it doesn't seem to be.

Comment (foolip) [reply LICENSE.md]: I've compared it again and the only difference is indeed "The".

I also compared to the license file that GitHub gives you if you create a new repo with a license, and the only non-whitespace difference was that they call it "BSD 3-Clause License".

I've pushed @jugglinmike's changes to this PR and restored the documentation link and intend to merge this PR once checks have passed.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Sorry for the noise on this PR, I pushed some changes from https://github.com/web-platform-tests/wpt/pull/11191 with the intention of merging this, but https://github.com/web-platform-tests/rfcs/pull/18 references https://github.com/web-platform-tests/wpt/pull/11191 so I've undone that. This PR is pointing to the same commit as when it was created.

**Comment by foolip** [comment]
This has effectively been merged via https://github.com/web-platform-tests/wpt/pull/11191, which was almost identical apart from a "The" and whitespace, which is now the cause of conflicts. I'm going to close this, but thank you @wseltzer for creating it to make it clear that you were OK with this change.

**Pull request closed without merging**